app/controllers/assessments_controller.rb

```diff
@@ -64,6 +64,8 @@ class AssessmentsController < ApplicationController
   action_auth_level :set_repo, :instructor
   action_auth_level :import_svn, :instructor
 
+  protect_from_forgery with: :exception
```

Comment:
Would be good to figure out why `protect_from_forgery` inside `application_controller.rb` isn't sufficient